package/gluon-setup-mode/files/lib/gluon/setup-mode/rc.d/S96led

```diff
@@ -4,8 +4,13 @@ START=96
 
 start() {
 	/etc/init.d/led start
-
 	. /etc/diag.sh
-	get_status_led
+	. /lib/functions/leds.sh
+	if [ -e /proc/device-tree/aliases/led-boot ]
+	then
+		status_led="$(get_dt_led boot)"
+	else
+		get_status_led
+	fi
```

Comment:
AFAICT, get_status_led is defined just like this on ipq40xx, no need to change anything here.

Reply:
It is defined in master but not in release: https://github.com/openwrt/openwrt/commit/4fc005197a320ff728544eba8028f7a9f23e1868

Reply:
Fixed in OpenWrt, will update in Gluon soon.